src/rest-server/docs/swagger.yaml

```diff
@@ -2655,6 +2839,144 @@ components:
                 - taskStatuses
         isLatest:
           type: boolean
+    TaskStatus:
+      type: object
+      description: task status
+      properties:
+        taskIndex:
+          type: integer
+          description: task index
+        taskUid:
+          type: string
+        taskState:
+          type: string
+          description: task state
+          enum:
+            - WAITING
+            - RUNNING
+            - STOPPING
+            - STOPPED
+            - SUCCEEDED
+            - FAILED
+            - UNKNOWN
+        retries:
+          type: integer
+        accountableRetries:
+          type: integer
+        createdTime:
+          type: integer
+          nullable: true
+          description: >-
+            task created time, in number of milliseconds since the Unix Epoch.
+        completedTime:
+          type: integer
+          nullable: true
+          description: >-
+            task completion time, in number of milliseconds since the Unix Epoch.
+        username:
+          type: string
+        jobName:
+          type: string
+        jobAttemptId:
+          type: integer
+        taskRoleName:
+          type: string
+        attempts:
+          type: array
+          items:
+            type: object
+            description: task attempt specific information
+            properties:
+              attemptId:
+                type: integer
+              attemptState:
+                type: string
+              currentAttemptLaunchedTime:
+                type: integer
+                nullable: true
+                description: >-
+                  the current attempt launched time, in number of milliseconds since the Unix
+                  Epoch.
+              currentAttemptCompletedTime:
+                type: integer
+                nullable: true
+                description: >-
+                  the current attempt completion time, in number of milliseconds since the Unix
+                  Epoch.
```

Comment:
Seems no need "currentAttempt" prefix.
Just rename to launchedTime and completedTime?
And notify mingliang

Reply:
> Seems no need "currentAttempt" prefix.
> Just rename to launchedTime and completedTime?
> And notify mingliang

This is aligned with get job status api, maybe we can rename these fields together in the future ?